### PR TITLE
Issue 18591

### PR DESCRIPTION
**Pull request by wezell**
This fixes the escalationthread for multi-lingual contents

### CODE REVIEW

File: dotCMS/src/main/java/com/dotcms/workflow/EscalationThread.java [hunk 1]
```diff
@@ -36,11 +35,11 @@ public void run(final JobExecutionContext jobContext) throws JobExecutionExcepti
 
             try {
 
-                startTransaction();
+
                 final List<WorkflowTask> tasks = workflowAPI.findExpiredTasks();
 
                 for (final WorkflowTask task : tasks) {
-
+                    startTransaction();
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotcms/workflow/EscalationThread.java:42]: I would enclosed the body into a method and wrap into the transaction

File: dotCMS/src/main/java/com/dotcms/workflow/EscalationThread.java [hunk 2]
```diff
@@ -51,10 +50,10 @@ public void run(final JobExecutionContext jobContext) throws JobExecutionExcepti
                             "timeout on step '" + step.getName() + "' " +
                             "excecuting escalation action '" + action.getName() + "'");
 
-                    // find contentlet for default language
+                    // find contentlet for the given language
                     final Contentlet contentletByDefaultLanguage =
                             APILocator.getContentletAPI().findContentletByIdentifier(task.getWebasset(), false,
-                                APILocator.getLanguageAPI().getDefaultLanguage().getId(),
+                                task.getLanguageId(),
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotcms/workflow/EscalationThread.java:56]: cool

**Review by freddyucv — APPROVED** [review]

**Review by dsilvam — APPROVED** [review]

**Review by erickgonzalez — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/workflow/EscalationThread.java [hunk 3]
```diff
@@ -1,12 +1,18 @@
 package com.dotcms.workflow;
 
+import static com.dotmarketing.db.HibernateUtil.closeSessionSilently;
+import static com.dotmarketing.db.HibernateUtil.rollbackTransaction;
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/workflow/EscalationThread.java:4]: Codacy Issue found: [Avoid unused imports such as 'com.dotmarketing.db.HibernateUtil.rollbackTransaction'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5634360)

File: dotCMS/src/main/java/com/dotcms/workflow/EscalationThread.java [hunk 4]
```diff
@@ -1,12 +1,18 @@
 package com.dotcms.workflow;
 
+import static com.dotmarketing.db.HibernateUtil.closeSessionSilently;
```
Comment (dev-dotcms) [inline dotCMS/src/main/java/com/dotcms/workflow/EscalationThread.java:3]: Codacy Issue found: [Avoid unused imports such as 'com.dotmarketing.db.HibernateUtil.closeSessionSilently'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5634360)

**Review by dsilvam — APPROVED** [review]

**Pull request merged**